app/javascript/views/referrals/style.ts

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div
+`
+  width: 100%;
+`
+
+export const Container = styled.div
+`
+  width: 1200px;
```

Comment:
I don't think we should be setting this value. I think instead we should probably set a padding on the left and right. Similar to [this article on the matter](https://css-tricks.com/best-way-implement-wrapper-css/). 

I was also looking at how we implement the wrapping on the grid. If we remove this style and the `min-width` in the `main-content.scss` then we can apply the following style to the grid 
```ts
export const Grid = styled.div
`
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(22rem, auto));
  grid-gap: 1rem;
  margin-top: 1rem;
`
```

Which will allow our tiles to be responsive and go to the next row when there is not enough space anymore.

Reply:
This is a good solution 👍